### PR TITLE
DOCS-2122 API Abuse monitoring mode

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/about-wallarm/api-abuse-prevention.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ You can configure API Abuse Prevention to react to malicious bots in one of the
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-abuse-prevention.md:71]: the safe blocking description says about trying the module without bot blocking but actually monitoring is the mode that has this purpose or both of them - users will be confused

File: docs/latest/about-wallarm/api-abuse-prevention.md [hunk 2]
```diff
@@ -71,6 +71,8 @@ You can configure API Abuse Prevention to react to malicious bots in one of the
 
     If you use filtering node in the safe blocking [mode](../admin-en/configure-wallarm-mode.md), some attacks originating from graylisted IPs will be blocked. [Read more](../user-guides/ip-lists/graylist.md)
 
+* **Only monitor**: Wallarm will display the detected bot activity in the **Events** section but will add the bot's IP neither to deny- nor to graylist. From any bot related event in `Monitoring` status, you can quickly block the bot with the **Add source IP to denylist** button.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-abuse-prevention.md:74]: 1. "From any bot related event in `Monitoring` status, you can quickly block the bot with the **Add source IP to denylist** button." - how? is each event provided with such button? add link to the events docs and probaly mention it in the event user guide
1. in THE monitoring status

Comment (MaximBashurov) [reply docs/latest/about-wallarm/api-abuse-prevention.md:74]: "how?" ← "From such events details, you can quickly block the bot with the Add source IP to denylist button."

Comment (MaximBashurov) [reply docs/latest/about-wallarm/api-abuse-prevention.md:74]: "probaly mention it in the event user guide" - I think this is not required

File: docs/latest/about-wallarm/api-abuse-prevention.md [hunk 3]
```diff
@@ -65,11 +65,8 @@ There are three available levels:
 You can configure API Abuse Prevention to react to malicious bots in one of the following ways:
 
 * **Add to denylist**: Wallarm will [denylist](../user-guides/ip-lists/denylist.md) bots' IPs and block all traffic these IPs produce.
-* **Add to graylist**: this reaction allows you to try the API Abuse Prevention functionality avoiding bots' blocking.
-
-    Wallarm will register bots' attacks and put bots' IPs to the [graylists](../user-guides/ip-lists/denylist.md) but will not block traffic these IPs produce.
-
-    If you use filtering node in the safe blocking [mode](../admin-en/configure-wallarm-mode.md), some attacks originating from graylisted IPs will be blocked. [Read more](../user-guides/ip-lists/graylist.md)
+* **Add to graylist**: Wallarm will [graylist](../user-guides/ip-lists/graylist.md) bots' IPs and block only malicious requests originating from them.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-abuse-prevention.md:68]: malicious bot requests are also malicious requests but we do not block them

File: docs/latest/about-wallarm/api-abuse-prevention.md [hunk 4]
```diff
@@ -65,11 +65,8 @@ There are three available levels:
...
+* **Only monitor**: Wallarm will display the detected bot activity in the [**Events**](../user-guides/events/check-attack.md) section but will add the bot's IP neither to deny- nor to graylist. From such events details, you can quickly block the bot with the **Add source IP to denylist** button. The IP is added to the denylist forever, but in the **IP Lists** section you can delete it or change the time of staying in the list.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-abuse-prevention.md:69]: From such events details, you can quickly block the bot with the **Add source IP to denylist** button. The IP is added to the denylist forever, but in the **IP Lists** section you can delete it or change the time of staying in the list. - make it to be the 2nd paragraph

**Pull request merged**